kubernetes-deployment/maintainlib/common.py

```diff
@@ -79,12 +79,12 @@ def write_generated_file(generated_file, file_path):
 
 def sftp_paramiko(src, dst, filename, host_config):
```

Comment:
>host_config [](start = 38, length = 11)

I suggest you add a function to validate the format of host_config, and use the funcation here and in ssh_shell_paramiko